### PR TITLE
Add --speedTest to feed client CLI, and dryRun to /doc/v1

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review

### CODE REVIEW

**Comment by jonmv** [comment]
Or @freva :)))

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/HttpFeedClient.java [hunk 1]
```diff
@@ -217,13 +219,14 @@ static String encode(String raw) {
...
 
-    static String getQuery(OperationParameters params) {
+    static String getQuery(OperationParameters params, boolean speedTest) {
         StringJoiner query = new StringJoiner("&", "?", "").setEmptyValue("");
         if (params.createIfNonExistent()) query.add("create=true");
         params.testAndSetCondition().ifPresent(condition -> query.add("condition=" + encode(condition)));
         params.timeout().ifPresent(timeout -> query.add("timeout=" + timeout.toMillis() + "ms"));
         params.route().ifPresent(route -> query.add("route=" + encode(route)));
         params.tracelevel().ifPresent(tracelevel -> query.add("tracelevel=" + tracelevel));
+        if (speedTest) query.add("speedTest=true");
```
Comment (freva) [inline vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/HttpFeedClient.java:229]:
```suggestion
        if (speedTest) query.add("dryRun=true");
```

**Review by freva — APPROVED** [review]
Shouldn't we document in CLI help that this requires that the endpoint runs Vespa >= 8.35?

**Comment by jonmv** [comment]
Yes, thank you!

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**